pkg/resources/fluentd/deployment.go

```diff
@@ -131,7 +131,7 @@ func generateVolumeMounts(fluentd *loggingv1alpha1.Fluentd) (v []corev1.VolumeMo
 	if fluentd.Spec.TLS.Enabled {
 		tlsRelatedVolume := []corev1.VolumeMount{
 			{
-				Name:      "fluentd-tls",
```

Comment:
why is this necessary?

Reply:
@pepov just for uniformity, related to this line:
https://github.com/banzaicloud/logging-operator/pull/134/files#diff-e0cf039a346a2f831decd3a8364eb052R96
A bit OCD of me I would agree. I'll revert it.

Reply:
I was too picky perhaps, but thanks!